doc/unity.rst

```diff
@@ -323,6 +323,20 @@ The plug-in allows using arbitrary Unity meshes for MuJoCo collision.  At model
 <http://www.qhull.org/>`__ to create a convex hull of the mesh, and uses that for collisions.  Currently the computed
 convex hull is not visible in Unity, but we intend to expose it in future versions.
 
+Height fields
+_____________
+
+MuJoCo hfields are represented in Unity through terrain gameobjects. This allows the use of the terrain editing tools
+available in Unity to generate shapes for collisions with MuJoCo. When selecting hfield type in the Unity geom
+component, the right click context menu provides utility to add the corresponding Unity terrain to the scene. The data
+from the terrain is dynamically kept in sync with the simulation.
+
+MuJoCo plugins
+______________
+
+The current version of the Unity package does not support loading MJCF scenes that use MuJoCo plugins such as
+``elasticity``. Adding basic functionality to do this will be part of an upcoming release.
```

Comment:
Link to https://github.com/google-deepmind/mujoco/tree/main/plugin/elasticity#readme ?